Operating System Refs new indicator field

ChanochShayner opened the pull request:
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/33656
bloked by: https://github.com/demisto/etc/issues/33806
## Description
Add a new indicator field `Operating System Refs`.

ChanochShayner commented:
@ytauman Do we want it to be `shortText` type or `tagsSelect` type?

bakatzir commented on Packs/CommonTypes/IndicatorFields/indicatorfield-operating_system_refs.json:
```diff
@@ -0,0 +1,27 @@
+    "content": true,
+    "group": 2,
+    "hidden": false,
+    "associatedToAll": true,
```
isn't is supposed to be associated to a specific IOC?

ChanochShayner replied:
added

bakatzir replied:
please check that, I think it should be the details filed, which is, e.g `STIX Malware`

ChanochShayner replied:
This is a case where we do not have a complete list of options, so we thought (me and @ytauman) we should use `tags` (as in stix aliases).

ChanochShayner commented:
I also added to this PR a transfer of the `MITER ID` indicator field from the `MITREAttackv1` pack to `CommonTypes` and added a new type to it.
@bakatzir @ytauman @yuvalbenshalom FYI

bakatzir reviewed (commented):
That's ok, but let's make sure to do 2 things:
1. bump the mitre and commontypes packs, and update a mandatory dep in between mitre pack to commontypes.
2. add an rn note in the mitre pack about the braking change, and add that without installing the commontypes pack uptodate, it will not work.

ChanochShayner commented on Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md:
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Fields
+**MITRE ID**
+- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version x.x.x and onwards to continue working seamlessly with this pack.
```
```suggestion
- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version #TODO and onwards to continue working seamlessly with this pack.
```

bakatzir commented on Packs/FeedMitreAttack/pack_metadata.json:
```diff
@@ -16,5 +16,11 @@
         "Feed",
         "ATT&CK"
-    ]
+    ],
+    "dependencies": {
+        "GetLicenseID": {
+            "mandatory": true,
+            "name": "Get License ID"
+        }
+    }
```
why is this needed?

ChanochShayner replied:
fixed

amshamah419 commented on Packs/CommonTypes/ReleaseNotes/2_9_8.md:
```diff
@@ -0,0 +1,17 @@
+- **Operating System Refs**
+- **MITRE ID**
+
+- Associated the **Kill Chain Phases** field with the following *Attack Pattern* and *Tool* indicator types.
```
```suggestion
- Associated the **Kill Chain Phases** field with the *Attack Pattern* and *Tool* indicator types.
```

amshamah419 commented on Packs/FeedMitreAttack/ReleaseNotes/1_1_13.md:
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Fields
+**MITRE ID**
+- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to version 2.9.8 and onwards to continue working seamlessly with this pack.
```
```suggestion
- Moved the indicator field to the CommonTypes pack. Update the CommonTypes pack to minimum version 2.9.8 to continue working with this pack.
```
